src/main/java/org/openhab/binding/zwave/handler/ZWaveSerialHandler.java

```diff
@@ -199,7 +201,7 @@ private void sendResponse(int response) {
                     logger.trace("Response SENT {}", response);
                 }
             } catch (IOException e) {
-                logger.error("Exception during send", e);
+                logger.debug("Exception during send", e);
```

Comment:
Probably this should be WARN - this is quite a serious error as commands are not being sent.

Reply:
Changed to warn.